runtime/Python3/src/antlr4/_pygrun.py

```diff
@@ -1,10 +1,9 @@
-#!python
 __author__ = 'jszheng'
 import optparse
 import sys
 import os
-import importlib
-from antlr4 import *
+
+from . import *
```

Comment:
This is the only place where `import *` is not fixed - I would be happy to follow-up in another PR to make the names being used explicit.